src/api/concept.ts

```diff
@@ -0,0 +1,20 @@
+import useSWR from "swr";
+import { openmrsFetch, showToast } from "@openmrs/esm-framework";
+import { Concept } from "./types";
+
+export function useConcepts() {
+  const url = "/ws/rest/v1/concept?class=Question&v=full";
+  const { data, error } = useSWR<{ data: { results: Array<Concept> } }, Error>(
+    url,
+    openmrsFetch
+  );
+  if (error) {
+    showToast({
+      title: "Error",
+      kind: "error",
+      critical: true,
+      description: `${error.message}`,
+    });
+  }
+  return { concepts: data?.data?.results ?? [] };
```

Comment:
My thinking is that you are fetching or Question concepts and having those as options readily available in a combo box. Well for a system with a very bulky concept dictionary, this may not scale. @kumuditha-udayanga  about implementing a 'concept search' kind of approach were the user gets to search the concepts they need?